standard/expressions.md

```diff
@@ -4300,6 +4322,27 @@ A *throw expression* shall only occur in the following syntactic contexts:
 - As the second operand of a null coalescing operator (`??`).
 - As the body of an expression-bodied lambda or member.
 
+## §declaration-expressions-new-clause Declaration expressions
+
+A declaration expression declares a local variable.
+
+``` antlr
+declaration_expression
+    : local_variable_type identifier
+```
+
+A declaration expression shall only occur in the following syntactic contexts:
+
+- As an `out` *argument_value* in an *argument_list*.
+- As a *tuple_element* in a *tuple_expression* that occurs on the left-hand side of a deconstructing assignment.
+
+The *local_variable_type* of a *local_variable_declaration* either directly specifies the type of the variable introduced by the declaration, or indicates with the identifier `var` that the type is implicit and should be inferred based on the syntactic context as follows:
```

Comment:
Should *local_variable_declaration* here be *declaration_expression*?

Reply:
Yes. Thanks!

Reply:
Fixed!